aclk/agent_cloud_link.c

```diff
@@ -129,6 +161,31 @@ struct aclk_query_queue {
     u_int64_t count;
 } aclk_queue = { .aclk_query_head = NULL, .aclk_query_tail = NULL, .count = 0 };
 
+/*
+ * After a connection failure -- delay in milliseconds
+ * When a connection is established, the delay function
+ * should be called with mode 0 to reset the fail count
+ */
+unsigned long int aclk_delay(int mode)
+{
+    static int fail = 0;
```

Comment:
If fail is intended as a counter then I don't see where timeout failure causes it to be incremented?